index.html

```diff
@@ -116,34 +119,81 @@ <h3 class="year">2015</h3>
+                            <li class="skills">HTML</li>
+                            <li class="skills">CSS</li>
+                            <li class="skills">JAVASCRIPT</li>
+                        </ul>
+                    </div>
+                    <button class="action">See project</button>
+                </div>
+            </div>
+        
+    </section>
 
+    <!-- About -->
+    <section class="about-myself">
+        <div class="top">
+            <h1 class="top-heading">About Myself</h1>
+            <p class="top-para">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <div class="social-media">
+                <h3 class="social">LET’S CONNECT</h3>
+                <div class="icons2">
+                    <i class="fab fa-linkedin-in"></i>
+                    <i class="fab fa-angellist"></i>
+                    <i class="fab fa-twitter"></i>
+                    <i class="fab fa-medium-m"></i>
+                </div>
+            </div>
+            <button class="resume">Get my resume</button>
+        </div>
+        <div class="skill-list">
+            <div class="frame3">
+                <div class="frame4">
+                    <h2 class="list">Languages</h2>
+                    <i class="fas fa-chevron-down" style="color: #acb7c3;"></i>
+                </div>
+                <div class="lang-list">
+                    <div class="frame5">
+                        <i class="fab fa-js-square fa-lg" style="color: #ffff00;"></i>
+                        <h3 class="lang1">JavaScript</h3>
+                    </div>
+                    <div class="frame6">
+                        <i class="fab fa-html5 fa-lg" style="color: #ff5349;"></i>
+                        <h3 class="lang2">HTML</h3>
+                    </div>
+                    <div class="frame7">
+                        <i class="fab fa-css3-alt fa-lg" style="color: #0087ce;"></i>
+                        <h3 class="lang3">CSS</h3>
                     </div>
                 </div>
-                <p class="primary">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <div class="tags">
-                    <ul class="list">
-                        <li class="skills">HTML</li>
-                        <li class="skills">CSS</li>
-                        <li class="skills">JAVASCRIPT</li>
-                    </ul>
+                <div class="frame8">
+                    <h2 class="txt1">Frameworks</h2>
+                    <i class="fas fa-chevron-right" style="color: #acb7c3;"></i>
+                </div>
+                <div class="frame9">
+                    <h2 class="txt2">Skills</h2>
+                    <i class="fas fa-chevron-right" style="color: #acb7c3;"></i>
```

Comment:
Hello @cate339 👋 
Good job so far! 😃  However, I recommend that you use external CSS instead of inline styling. This is because external CSS provides better code organization, reusability, and maintainability. It allows you to separate the styling concerns from the HTML structure, making your code more modular and easier to update in the future. Additionally, external CSS files can be cached by the browser, resulting in faster page load times for subsequent visits to your website.